Lib/traceback.py

```diff
@@ -883,7 +896,12 @@ def format(self, *, chain=True, _ctx=None):
                     yield from _ctx.emit('Traceback (most recent call last):\n')
                     yield from _ctx.emit(exc.stack.format())
                 yield from _ctx.emit(exc.format_exception_only())
+            elif _ctx.exception_group_depth > self.max_group_depth:
+                # exception group, but depth exceeds limit
+                yield from _ctx.emit(
+                    f" ... (max_group_depth is {self.max_group_depth})\n")
```

Comment:
Nit: no leading space (there's already one due to the margin and it looks weird with two).